source/user-manual/ruleset/custom.rst

```diff
@@ -44,17 +44,14 @@ Check out this example on how to create new decoders and rules. The following lo
       </decoder>
 
 
-#. Add the following rule to ``/var/ossec/etc/rules/local_rules.xml``:
+#. Add the following rule to ``/var/ossec/etc/rules/local_rules.xml``. Place it within a ``<group>`` element, for example ``<group name="custom_rules_example,">``.
 
    .. code-block:: xml
 
-      <group name="custom_rules_example,">
-        <rule id="100010" level="0">
-          <program_name>example</program_name>
-          <description>User logged</description>
-        </rule>
-      </group>
-
+      <rule id="100010" level="0">
```

Comment:
I think it's a good idea to include a `<group>` element this example as in the original commit.